package/samplers/smac_sampler/sampler.py

```diff
@@ -226,6 +227,11 @@ def _get_init_design(
             raise NotImplementedError(f"Unknown Initial Design Type: {init_design_type}")
         return init_design
 
+    def reseed_rng(self) -> None:
```

Comment:
`reseed_rng` is currently not implemented. This will cause unexpected behavior when `n_jobs > 1`.